### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by user18081972** [rustdesk/rustdesk#8216] [General]

**Comment by anoraxx** [comment]
Almost: [Windows Portable Elevation](https://rustdesk.com/docs/en/client/windows-portable-elevation/)

The only thing that's missing from how TeamViewer works is that someone still has to confirm the UAC dialog while TeamViewer will restart its client with elevated privileges using the entered credentials. The credentials can also be sent from the controlling end though so that the person on the controlled end only has to do one click.
